docs/v3.x/concepts/templates.md

```diff
@@ -0,0 +1,79 @@
+# Templates
+
+A template is a pre-made Strapi configuration designed for a specific use case. It allows you to quickly boostrap a custom Strapi app.
+
+Here are some things a template may configure for you:
+
+- Collection types and single types
+- Components and dynamic zones
+- Plugins to install, or custom plugins
+
+::: warning
+Templates and starters are not the same thing:
+
+- A _template_ is a pre-made Strapi configuration
+- A _starter_ is a pre-made frontend application that consumes a Strapi API
+  :::
+
+## Using a template
+
+You can use a template when creating a project with `create-strapi-app`.
+
+:::: tabs
+
+::: tab yarn
+
+```bash
+yarn create strapi-app my-project --template <template-github-url>
+```
+
+:::
+
+::: tab npx
+
+```bash
+npx create-strapi-app my-project --template <template-github-url>
+```
+
+:::
+
+::::
+
+You can use the `--template` option in combination with all other `create-strapi-app` options, like `--quickstart` or `--no-run`.
```

Comment:
What about the database options?

Reply:
All current options, like `--quickstart` or `dbhost`, `dbname` and so on, should not be affected. This is why we're strict about the files a template can have